sonaranalyzer-dotnet/its/expected/SkipGeneratedVb/SkipGeneratedVb-{C4F122A1-A51D-4194-998F-686A6EF31E44}-S1172.json

```diff
@@ -0,0 +1,30 @@
+{
+"issues":  [
+{
+"id":  "S1172",
+"message":  "Remove this unused procedure parameter 'args'.",
+"location":  {
+"uri":  "sources\GeneratedCodeVb\NotIgnored.vb",
+"region":  {
+"startLine":  6,
+"startColumn":  35,
+"endLine":  6,
+"endColumn":  57
+}
+}
+},
+{
+"id":  "S1172",
+"message":  "Remove this unused procedure parameter 'args'.",
+"location":  {
+"uri":  "sources\GeneratedCodeVb\Region.vb",
```

Comment:
this is not skipped... the Region is `#Region "Degenerated code"` and is not skipped